### PR TITLE
Integrate 5/16 Nightly RN Build 

### PR DESCRIPTION
**Pull request by NickGerleman**
See original commits here https://github.com/facebook/react-native/compare/30a89f30c...6a96a9f65

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/5767)

### CODE REVIEW

File: vnext/ReactCopies/RNTester/js/components/TextInlineView.js [hunk 1]
```diff
@@ -25,6 +25,21 @@ function Basic(): React.Node {
...
 }
 
+function NestedTexts(): React.Node {
+  return (
+    <View>
+      <Text>This is the first row</Text>
+      <Text>
+        <Text>
+          <Text>This is a nested text </Text>
+          <View style={{height: 20, width: 20, backgroundColor: 'red'}} />
```
Comment (acoates-ms) [inline vnext/ReactCopies/RNTester/js/components/TextInlineView.js:35]: I'm assuming this doesn't work for windows right now?

Comment (NickGerleman) [reply vnext/ReactCopies/RNTester/js/components/TextInlineView.js:35]: Yeah, this was only added to the Android TextExample, so we don't ever see this added to our tester, but IIRC we YellowBox on nested text elements. I think we have an issue somewhere about nested text but am having trouble finding it.

**Comment by TheSavior** [comment]
I really like seeing these PRs, you are making amazing progress. 🎉

As a meta note, I think it would be nice if you could include the changelog (including internal commits) in your PR summaries for these. The link to the commits is good, but as a casual observer having them grouped and categorized would make it easier to kinda scan and comment on the challenges or flag things in those commits that I could imagine being missed.

**Comment by NickGerleman** [comment]
> As a meta note, I think it would be nice if you could include the changelog (including internal commits) in your PR summaries for these. The link to the commits is good, but as a casual observer having them grouped and categorized would make it easier to kinda scan and comment on the challenges or flag things in those commits that I could imagine being missed.

Definitely agree with this. I have some work ahead of me to automated PR creation at shorter intervals and add some more signals. In the short term I just now discovered the RN changelog generation script lets you pass arbitrary commits https://github.com/react-native-community/releases/blob/master/scripts/changelog-generator.ts. Will use that in the future.

File: vnext/PropertySheets/React.Cpp.props [hunk 2]
```diff
@@ -111,6 +111,17 @@
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
       <RuntimeTypeInfo>false</RuntimeTypeInfo>
       <AdditionalOptions>/utf-8 %(AdditionalOptions)</AdditionalOptions>
+
+      <!-- Add include directories expected by React Native C++ code and headers -->
```
Comment (dannyvv) [inline vnext/PropertySheets/React.Cpp.props:115]: Thank you!!!

**Comment by NickGerleman** [comment]
Talked to @tudorms and @acoates-ms more offline about JSI headers. We will likely want a fix in the next couple weeks to prevent losing coverage, but can disable non-Chakra engines for now. Likely want to branch/version the runtimes we're consuming.

**Pull request merged**